visualization/include/pcl/visualization/vtk.h

```diff
@@ -1,172 +0,0 @@
-/*
```

Comment:
Awesome deletion :heart: 

But please replace with deprecated pragma macro (but still remove the contents since we don't offer header include stability guarantees)